src/olympia/reviewers/forms.py

```diff
@@ -149,8 +149,8 @@ def create_option(self, *args, **kwargs):
             # Add our special `data-toggle` class and the right `data-value` depending
             # on status.
             option['attrs']['class'] = 'data-toggle'
-            option['attrs']['data-value'] = '|'.join(
-                self.actions_filters[obj.channel].get(status, ()) + ('',)
+            option['attrs']['data-value'] = ''.join(
+                f' {a} ' for a in self.actions_filters[obj.channel].get(status, ())
```

Comment:
As mentioned on matrix, if you use `~=` without adding spaces in the JS (*) then you can drop the extra whitespace before and after each value here and below, joining with a space instead. It would look much better and will be less confusing IMHO.

(*) Assuming jQuery supports `~=`. That needs to be double-checked.

Reply:
https://github.com/mozilla/addons-server/pull/19894/commits/8c59f0a4651974249f6224a9af19d960670b4770